ifrs17/Import/ImportStorage.ipynb

```diff
@@ -369,7 +369,11 @@
                 "\n    ",
                 "\n    public double GetPremiumAllocationFactor(ImportIdentity id) => SingleDataNodeParametersByGoc.TryGetValue(id.DataNode, out var singleDataNodeParameter) ",
                 "\n                                                                                ? singleDataNodeParameter[CurrentPeriod].PremiumAllocation : DefaultPremiumExperienceAdjustmentFactor;",
-                "\n    ",
+                "\n",
+                "\n    public double[] GetAmortizationFactor(ImportIdentity id) => SingleDataNodeParametersByGoc.TryGetValue(id.DataNode, out var singleDataNodeParameter) ",
```

Comment:
this getter should change. 
input is the identity tuple (with amountype, ...)
the body of the method does: 
1. checks if RawVariable contains such a pattern, if yes return 
2. if not, checks if SingleDataNodeParameter has a pattern, if yes return
3. if not, log error: MissingPattern for goc "X"